src/components/tag.scss

```diff
@@ -0,0 +1,87 @@
+%tag-global {
+  // Public API (allowed to be overridden)
+  --_op-tag-background-color: var(--op-color-primary-base);
```

Comment:
Probably can follow badge and button and simplify the API (or remove it altogether)